ct/ersatztv.sh

```diff
@@ -26,29 +26,40 @@ function update_script() {
         msg_error "No ${APP} Installation Found!"
         exit
     fi
-
+    RELEASE=$(curl -s https://api.github.com/repos/ErsatzTV/ErsatzTV/releases | grep -oP '"tag_name": "\K[^"]+' | head -n 1)
+    if [[ ! -f /opt/${APP}_version.txt && $(echo "x.x.x" > /opt/${APP}_version.txt) || "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]]; then
     msg_info "Stopping ErsatzTV"
     systemctl stop ersatzTV
     msg_ok "Stopped ErsatzTV"
 
     msg_info "Updating ErsatzTV"
-    RELEASE=$(curl -s https://api.github.com/repos/ErsatzTV/ErsatzTV/releases | grep -oP '"tag_name": "\K[^"]+' | head -n 1)
     cp -R /opt/ErsatzTV/ ErsatzTV-backup
     rm ErsatzTV-backup/ErsatzTV
     rm -rf /opt/ErsatzTV
-    wget -qO- "https://github.com/ErsatzTV/ErsatzTV/releases/download/${RELEASE}/ErsatzTV-${RELEASE}-linux-x64.tar.gz" | tar -xz -C /opt
-    mv "/opt/ErsatzTV-${RELEASE}-linux-x64" /opt/ErsatzTV
+    temp_file=$(mktemp)
+    wget -qO- https://github.com/ErsatzTV/ErsatzTV/releases/download/${RELEASE}/ErsatzTV-${RELEASE}-linux-x64.tar.gz -O "$temp_file"
+    tar -xzf "$temp_file"
+    mv /opt/ErsatzTV-${RELEASE}-linux-x64 /opt/ErsatzTV
```

Comment:
This is a breaking change because it assumes a person is in /opt when they run `update`.  I'd suggest removing "/opt/" from the source so it will use the CWD